submodules/LIB/hardware/syn/yosys/build.tcl

```diff
@@ -0,0 +1,13 @@
+ 
+
+# Read the Verilog files
+
+yosys read_verilog -I./src -I../src -defer ../src/*.vh ../src/*.v
```

Comment:
Again:

../src/*.vh is not needed try without  (please learn what is a header search path)

what is -defer ?

Reply:
-defer

only read the abstract syntax tree and defer actual compilation
to a later 'hierarchy' command. Useful in cases where the default
parameters of modules yield invalid or not synthesizable code.

This is from yosys manual. originally yosys wanst reading the verilog code well. so this option was implemented in order for it to not have those mistakes. but i will test if it can be sucessefully run without that command.